Makefile

```diff
@@ -142,6 +157,52 @@ lint: ## Download & Build & Run golangci-lint against code.
 	GOBIN=$(LOCALBIN) go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANG_CI_LINT_VERSION)
 	$(LOCALBIN)/golangci-lint run
 
+.PHONY: configure-git-origin
+configure-git-origin:
+	@git remote | grep '^origin$$' -q || \
+		git remote add origin https://github.com/kyma-project/eventing-manager
```

Comment:
why `eventing-manager`?

Reply:
Wow, oversaw this. It's copied from eventing-manager, left-over. Fixed